test/serializer/optimized-functions/ObjectAssign.js

```diff
@@ -1,4 +1,4 @@
-// Copies of assign\(:1
+// Copies of assign\(:0
```

Comment:
Why is this now expected?

Reply:
Because we now inline it, we no longer need to emit an Object.assign.

Reply:
It is not clear to me what is being inlined here and why it is OK for Object.assign to go away.

Reply:
Sorry, I misread which file this commented related to. I've updated the test comment.